docs/en/Processors.md

```diff
@@ -13,3 +13,9 @@ The `DataListProcessor` class is a basic processor which is used to delete a Lis
 The `SQLExpressionProcessor` class is more advanced processor which is used execute SQLDelete queries against the Database. It will convert any `SQLExpression` within a collection into an `SQLDelete` statement and process is.
 
 **Implementor Class:** `SilverStripe\ORM\Queries\SQLExpression`
+
+## RawSQL Processor
+
+The `RawSQLProcessor` class is a basic processor which i used to execute raw SQL directly on the SilverStripe Application's Database. A Custom Model is provided so that Implementor Class matching can occur on the provided query.
```

Comment:
```suggestion
The `RawSQLProcessor` class is a basic processor which is used to execute raw SQL directly on the SilverStripe Application's Database. A Custom Model is provided so that Implementor Class matching can occur on the provided query.
```